### ISSUE TITLE
Comparing PointRCNN performance to baseline

### ISSUE THREAD

**Issue by baraujo98** [facebookresearch/DepthContrast#16]
Hi @zaiweizhang! This is a minor question related to the paper. I did my own baseline training of PointRCNN (with no pretraining) and achieved better results than those you presented in the paper, namely in the `car` and `cyclist` classes. The only thing I changed from the original config was the batchsize per GPU (to 10), used 1 GPU and tuned the lr accordingly (to original_lr * 10 / 24 = 0.0084).

Doing a median of the moderate mAP for the last 11 epochs for the baseline, and trying to read the results from the paper's graphs I get:

|         | My baseline | Paper's baseline | Paper finetuning results |
|:-------:|:-----------:|:----------------:|:------------------------:|
|     Car |     80,5    |       ~ 80       |           ~ 80           |
| Cyclist |     72,6    |      ~ 70,3      |          ~ 72,5          |
|     Ped |     53,1    |      ~ 57,3      |          ~ 57,7          |

So, do you think the fact that the baseline I trained is better than what you reported can be due to the batchsize difference?Interestingly enough, `pedestrian` got a worse result in my baseline than in yours. Is your baseline the trained model supplied in OpenPCDet or did you train it?

**Comment by baraujo98** [comment]
Thanks. Interesting, was there any reason behind only using xyz and not intensity? Is this a common practice of some sort?

I'm also planning on training some baselines with limited labels in the same conditions and comparing to pretrained+finetuned models.

**Comment by zaiweizhang** [comment]
No. We just think that maybe not all datasets contain intensity. Only using xyz may be generalized to any other task.

**Comment by baraujo98** [comment]
Ok. As much as I can tell, I think your fork of pcdet does use xyz and reflectance, right?

**Comment by zaiweizhang** [comment]
Yeah. When you are pretraining, did you use reflectance? Have you met any issues when loading the pretrained models?

**Comment by baraujo98** [comment]
Yes, I think the pretraining uses reflectance by default. I have met no problems so far.
